docs/installation.md

```diff
@@ -388,87 +335,87 @@ Get the logs from that pod:
 kubectl logs POD_NAME -f
 ```
 
-### Prometheus Troubleshooting
-
-Please refer to [Troubleshooting section in Prometheus document](/docs/prometheus.md#troubleshooting).
-
-## Upgrading Sumo Logic Collection
-
-To upgrade our helm chart to a newer version, you must first run update your local helm repo.
-
-```bash
-helm repo update
-```
-
-Next, you can run `helm upgrade --install` to upgrade to that version. The following upgrades the current version of `my-release` to the
-latest.
+## Installing in OpenShift platform
 
-```bash
-helm upgrade --install my-release sumologic/sumologic -f `user-values.yaml`
-```
+The daemonset/statefulset fails to create the pods in Openshift environment due to the request of elevated privileges, like HostPath mounts,
+privileged: true, etc.
 
-If you wish to upgrade to a specific version, you can use the `--version` flag.
+If you wish to install the chart in the Openshift Platform, it requires a SCC resource which is only created in Openshift (detected via API
+capabilities in the chart), you can add the following configuration to `user-values.yaml`:
 
-```bash
-helm upgrade --install my-release sumologic/sumologic -f `user-values.yaml` --version=2.0.0
+```yaml
+sumologic:
+  scc:
+    create: true
+otellogs:
+  daemonset:
+    containers:
+      otelcol:
+        securityContext:
+          privileged: true
```

Comment:
is this privilege elevation only required in openshift?

Reply:
I believe so, but I'm not an expert on OpenShift. @kkujawa-sumo do you know?